README.md

```diff
@@ -1,17 +1,44 @@
-# conda-dot-org
+# Conda.org
 
-This is the repository for the conda.org community website. To become involved:
+Welcome to the repository for the Conda.org community website. This is the central hub for the Conda community and serves as a platform for sharing information, resources, and discussions related to Conda.
 
-* Join the [conda.org chat room](https://app.element.io/#/room/#conda.org:matrix.org), which is part of the [conda space on Matrix](https://app.element.io/#/room/#conda:matrix.org)
-* Visit the [Root HackMD page](https://hackmd.io/DGtozSlsSjSokpYAK5-9hw), which links to everything in HackMD.
-* We meet every two weeks on Monday at 9am US Central time / 16:00 Central European Time. Please join the [Matrix room](https://app.element.io/#/room/#conda.org:matrix.org) and we will invite you.
+## Getting involved
 
-## Code of Conduct
+To become involved with the Conda community and contribute to the website, there are a few steps you can follow:
+
+- Join the [conda.org chat room](https://app.element.io/#/room/#conda.org:matrix.org), which is part of the [conda space on Matrix](https://app.element.io/#/room/#conda:matrix.org)
+- Visit the [Root HackMD page](https://hackmd.io/DGtozSlsSjSokpYAK5-9hw), which links to everything in HackMD.
+- Attend our bi-weekly meetings, which are held on Mondays at 9am US Central time / 16:00 Central European Time. - - - Please join the [Matrix room](https://app.element.io/#/room/#conda.org:matrix.org) and we will invite you.
 
-This repo and the web site it generates are governed by the [conda organization code of conduct](CODE_OF_CONDUCT.md).
 
 ## Implementation details
```

Comment:
I think `## Implementation Details` can be shifted to [CONTRIBUTING.md](https://github.com/conda-incubator/conda-dot-org/blob/main/CONTRIBUTING.md) file.